python/mxnet/text/embedding.py

```diff
@@ -88,18 +88,24 @@ def __init__(self, **kwargs):
         super(TokenEmbedding, self).__init__(**kwargs)
 
     @classmethod
-    def _get_pretrained_file_path_from_url(cls, url, embedding_root,
-                                           pretrained_file_name):
-        """Get the local path to the pre-trained token embedding file from url.
+    def _get_download_file_name(cls, pretrained_file_name):
```

Comment:
It looks that argument pretrained_file_name is returned. Why do we need this method?

Reply:
This is the mapping from the file-to-load to the file-to-download. Here in the base class, it assumes those two are the same. This function is overridden in GloVe because files are in zipped archives.